src/mlbstandings/web.py

```
@@ -7,21 +7,31 @@
+        elif isinstance(e, urllib.error.HTTPError):
+            code = e.code
+        else:
+            return False
+        return not Web.retryable(code)
     @staticmethod
     @backoff.on_exception(
         backoff.expo,
         (requests.exceptions.HTTPError, urllib.error.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout),
         max_time=600,
-        giveup=lambda e : isinstance(e, requests.exceptions.HTTPError) and e.response.status_code not in set([429, 500, 503]),
+        giveup=lambda e : Web.giveup(e),
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)